db/migrate/20200622221335_remove_lang_from_projects.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveLangFromProjects < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :projects, :lang
```

Comment:
don't you need this data before dropping the column?

Reply:
Not really, we never set it